README.md

```diff
@@ -1,15 +1,15 @@
 # Scaffold for social media app with Ruby on Rails
 
-> This repo includes intial code for social media app with basic styling. Its purpose is to be a starting point for Microverse students.
+> This repo includes intial code for social media app with basic styling of the ERD, a relationship diagram which signifies how the different entities and the various entity keys are related. It is contained in the doc folder. 
 
```

Comment:
Please update the Readme with the information about the current status of the app and what it does.